.github/workflows/trunk-cd.yml

```diff
@@ -21,8 +19,29 @@ jobs:
       - run: yarn --frozen-lockfile
       - run: yarn -s test:unit
 
+  system-tests:
```

Comment:
On trunk we system test before publishing, to increase confidence in our previous